README.md

```diff
@@ -1,3 +1,27 @@
-# Plain HTML Pages Template
+# SSE-Notes
 
-This is a template that uses plain HTML to create a static web page that is published at git-pages.thm.de.
+This is a project for our Secure Software Engineering course.
+
+It consists of a server and a client which make the writing and saving of private and public markdown notes possible.
+
+## Current State Of The Project
+[![CodeQL](https://github.com/BranAndSceolan/SSE-Notes/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/BranAndSceolan/SSE-Notes/actions/workflows/codeql-analysis.yml)
+
+## Building This Project
+
+### Backend
+* go into the backend directory ``cd backend``
+* make sure all dependencies are installed ``npm install``
+* either:
+  * build for development: 
+    * create a .env file with
+      * NOTES_PASSWORD=choose_a_password
+      * POSTGRES_USER=choose_an_admin_name
+      * POSTGRES_PASSWORD=choose_an_admin_password
+      * POSTGRES_DB=choose_a_db_name
+      * NODE_ENV=debug
+      * NOTES_USER=choose_a_username
+    * Replace all choose_-entries with the values you would like to use.
+    * source your .env file  ``source <path to .env>``
```

Comment:
This is an incomplete command, is it a parameter?

Reply:
yes, <path to .env> is intended as a placeholder for a parameter here.